daemon/src/model/cfd.rs

```diff
@@ -1234,20 +1234,31 @@ pub fn calculate_long_liquidation_price(leverage: Leverage, price: Price) -> Pri
     price * leverage / (leverage + 1)
 }
 
-/// Returns the Profit/Loss (P/L) as Bitcoin. Losses are capped by the provided margin
 pub fn calculate_profit(
+    payout: SignedAmount,
+    margin: SignedAmount,
+) -> Result<(SignedAmount, Percent)> {
+    let profit = payout - margin;
+    let percent = Decimal::from_f64(100. * profit.as_sat() as f64 / margin.as_sat() as f64)
+        .context("Unable to compute percent")?;
+
+    Ok((profit, Percent(percent)))
+}
+
+/// Returns the Profit/Loss (P/L) as Bitcoin. Losses are capped by the provided margin
+pub fn calculate_profit_from_current_price(
     initial_price: Price,
```

Comment:
I think we call this `opening_price` in many other places.

Reply:
this is not part of the changeset of this PR and it is called `initial_price` in `projection::Cfd` and `model::Cfd` so I will not change that here.